pkg/util/testutil/udp.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package testutil
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+)
+
+// GetAvailableUDPPort requests a random port number and makes sure it is available
+func GetAvailableUDPPort() (int, error) {
```

Comment:
```suggestion
func RandomUDPPort() (int, error) {
```
or
```suggestion
func UDPPort() (int, error) {
```
or
```suggestion
func FreeUDPPort() (int, error) {
```
or
```suggestion
func AvailableUDPPort() (int, error) {
```
Please don't use `Get*` (https://golang.org/doc/effective_go#Getters)

Reply:
The function `GetAvailableUDPPort` was moved and I have kept the same name.
https://golang.org/doc/effective_go#Getters dont' say to not use `Get*` for a function name but only to not use Get* for a getter.
If you prefer I can rename the function of course.

Reply:
It's fine Olivier, I don't have strong feelings. It was just a habit of mine to avoid Get*, but feel free to do as you wish with regards to this comment.